src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import React from "react";
+import { todo} from "../types";
+import classNames from "classnames";
+import { UserInfo } from "../UserInfo";
```

Comment:
The `UserInfo` component should not be imported directly here. Instead, the user data should be passed down as a prop from the parent component to maintain data consistency and follow best practices. Consider refactoring to receive the user data as a prop.